en/developer_guide/manage_assets.rst

```diff
@@ -0,0 +1,274 @@
+Manage assets with grunt
+========================
+
+Context
+-------
+
+To automate all the assets management process, the Open Orchestra Back Office uses the task manager
+`Grunt`_.
+
+*Grunt* is looking for its configuration inside the ``Gruntfile.js`` file in the root directory of
+the project. This file mainly parses external configuration files, split in grunt tasks and grunt
+tasks options.
```

Comment:
i think it's too early to talk about the specifics of orchestra : the splitting of the configuration inside grunt tasks and options among the different bundles should be explained latter